Simplenote/WPAuthWindowController.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  WPAuthViewController.swift
+//  Simplenote
+//
+
+import Cocoa
+import WebKit
+
+class WPAuthWindowController: NSWindowController {
+    
+    @IBOutlet var webView: WebView!
+    
+    @objc public func loadUrl(url: URL) {
+        webView.mainFrame.load(URLRequest.init(url: url));
```

Comment:
No need to explicitly call the `.init` + extra `;` at the end 😄 

```
webView.mainFrame.load(URLRequest(url: url))
```

Reply:
Haha those semicolons will never leave me I guess.